rust/tvm-macros/src/object.rs

```diff
@@ -75,6 +79,12 @@ pub fn macro_impl(input: proc_macro::TokenStream) -> TokenStream {
         _ => panic!("derive only works for structs"),
     };
 
+    let ref_derives = if derive {
```

Comment:
This is like if you mark something with a `no_derive` attribute you don't derive `Debug`? Naming is a bit wonky here

Reply:
yeah currently I just turn off all deriving for types marked no-derive (this all to support String which is like complete bespoke impl)